.github/scripts/install.sh

```diff
@@ -38,8 +38,9 @@ else
     # Install qtconsole from Github
     pip install git+https://github.com/jupyter/qtconsole.git
 
-    # Pin QtAwesome to 0.7.3 until we can update to 1.0.0
-    pip install qtawesome==0.7.3
+    # Install QtAwesome from Github
+    # pip install git+https://github.com/spyder-ide/qtawesome.git
+    pip install git+https://github.com/dalthviz/qtawesome.git@mdi_remap
```

Comment:
Now that https://github.com/spyder-ide/qtawesome/pull/155 is merged, you can change this to point to the main QtAwesome repo.